src/UglyToad.PdfPig/Export/PageXmlTextExporter.cs

```diff
@@ -0,0 +1,9820 @@
+                TextStyle = new PageXmlDocument.PageXmlTextStyle()
+                {
+                    FontSize = (float)letter.FontSize,
+                    FontFamily = letter.FontName,
+                    TextColourRgb = ToRgbEncoded(letter.Color),
+                },
+                TextEquivs = new[] { new PageXmlDocument.PageXmlTextEquiv() { Unicode = letter.Value } },
+                Id = "c" + glyphCount
+            };
+        }
+
+        private static PageXmlDocument Deserialize(string xmlPath)
+        {
+            XmlSerializer serializer = new XmlSerializer(typeof(PageXmlDocument));
+
+            using (var reader = System.Xml.XmlReader.Create(xmlPath))
+            {
+                return (PageXmlDocument)serializer.Deserialize(reader);
+            }
+        }
+
+        private string Serialize(PageXmlDocument pageXmlDocument)
+        {
+            XmlSerializer serializer = new XmlSerializer(typeof(PageXmlDocument));
+            var settings = new System.Xml.XmlWriterSettings()
+            {
+                //Encoding = new System.Text.UTF8Encoding(true),
+                Indent = true,
+                IndentChars = indentChar,
+                OmitXmlDeclaration = true // hack to manually handle utf-8
+            };
+
+            using (var stringWriter = new System.IO.StringWriter())
+            using (var xmlWriter = System.Xml.XmlWriter.Create(stringWriter, settings))
+            {
+                stringWriter.WriteLine("<?xml version=\"1.0\" encoding=\"UTF-8\"?>"); // hack to manually handle utf-8
+                serializer.Serialize(xmlWriter, pageXmlDocument);
+                return stringWriter.ToString();
+            }
+        }
+    }
+
+    #region PageXml Schema
+    /******************************************************************************
+     * PAGE pagecontent version 2019-07-15 https://www.primaresearch.org/schema/PAGE/gts/pagecontent/2019-07-15/pagecontent.xsd
+     * Auto-generated by xsd and improved by BobLD
+     ******************************************************************************/
+
+    /// <summary>
+    /// PAGE (Page Analysis and Ground-Truth Elements) root
+    /// <para>Version 2019-07-15</para>
+    /// </summary>
+    [EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)]
+    [GeneratedCodeAttribute("xsd", "4.6.1055.0")]
+    [SerializableAttribute()]
+    [DebuggerStepThroughAttribute()]
+    [DesignerCategoryAttribute("code")]
+    [XmlTypeAttribute(Namespace = "http://schema.primaresearch.org/PAGE/gts/pagecontent/2019-07-15")]
+    [XmlRootAttribute("PcGts", Namespace = "http://schema.primaresearch.org/PAGE/gts/pagecontent/2019-07-15", IsNullable = false)]
+    public class PageXmlDocument
+    {
+        private PageXmlMetadata metadataField;
+
+        private PageXmlPage pageField;
+
+        private string pcGtsIdField;
+
+        /// <remarks/>
+        public PageXmlMetadata Metadata
```

Comment:
I'm just going to try forking this branch and using Resharper to auto-auto-property these or it's going to take forever 😆